auctionapp/auction-app
# Partially complete Item component (minus message box rendering), complete SearchBar component

#### What's this PR do?

Completed most of the Item component except for the message box rendering and completed the SearchBar component. Note: Item component is taking in ID #3 for testing purposes - this will need to change once search results page is populated. For the search bar, the query currently takes exact searches - need to change this to include titles that contain the search as well.
#### Where should the reviewer start?

Item: client/components/Item.jsx, client/reducers/getItemData.js
SearchBar: client/components/SearchBar.jsx, client/reducers/searchItems.js
#### How should this be manually tested?

Create an item and click on the item portion of the navbar
Search for an item and check the store to see if the correct items have come through
#### Any background context you want to provide?

No
#### What are the relevant tickets?

n/a
